dev-tools/mage/crossbuild.go

```diff
@@ -193,8 +193,8 @@ func crossBuildImage(platform string) (string, error) {
 		tagSuffix = "s390x"
 	case strings.HasPrefix(platform, "linux"):
 		// Use an older version of libc to gain greater OS compatibility.
-		// Debian 7 uses glibc 2.13.
-		tagSuffix = "main-debian7"
+		// Debian 8 uses glibc 2.19.
+		tagSuffix = "main-debian8"
```

Comment:
in order to keep compatibility with glibc 2.13 we should keep using debian7 to build i386/amd64 binaries

Reply:
2.19 might be fine. See glibc compliance tracker: https://abi-laboratory.pro/?view=timeline&l=glibc

If not we might see beats-tester complain if we test with debian 7.

Reply:
```suggestion
		tagSuffix = "main-debian9"
```